components/remote_executor/remote_executor.py

```diff
@@ -22,18 +22,19 @@
         "user": user_id,
         "notebook": component_id,
         "polygon": area,
-        "date_from": start_date if start_date else '',
-        "date_to": end_date if end_date else ''
+        "date_from": start_date if start_date else "",
+        "date_to": end_date if end_date else "",
     }
     if additional_parameter_name:
         data["additional_parameter"] = additional_parameter_value
 
     request = geoapp_client.create_request(data=data)
     is_success, error_message = geoapp_client.wait_for_request_success_finish(
-        request.get('id'))
+        request.get("id")
+    )
     if not is_success:
         raise Exception(error_message)
-    paths_to_results = geoapp_client.pull_results(request.get("id"))
+    files_amount, paths_to_results = geoapp_client.pull_results(request.get("id"))
```

Comment:
Do we really need to return len(paths_to_results)? Isn't list is enough to know the number of results?